cfme/tests/infrastructure/test_providers.py

```diff
@@ -228,7 +228,7 @@ def test_provider_crud(provider):
 @test_requirements.provider_discovery
 @pytest.mark.parametrize('verify_tls', [False, True], ids=['no_tls', 'tls'])
 @pytest.mark.uncollectif(lambda provider:
-    not provider.one_of(RHEVMProvider) and not provider.data.get('verify_tls'))
+    not (provider.one_of(RHEVMProvider) and provider.endpoints.get('default').verify_tls))
```

Comment:
@slaykovsky Alexey, what will happen if you run collection with a provider that has no `verify_tls` under default endpoint? :wink: 

`AttributeError`.

I would strongly suggest running any rhev-only tests against some other provider as well. Maybe not locally, but certainly so on PRT. If you invoke `py.test` without `--use-provider`, it will run the `default` group which contains some VMWare providers. That will be sufficient.